frontend/src/metabase/embedding-sdk/components/SimpleDataPicker/SimpleDataPicker.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import { Icon, NavLink, Paper, ScrollArea, TextInput } from "metabase/ui";
+import type { TableId } from "metabase-types/api";
+
+interface SimpleDataPickerProps {
+  selectedEntity?: TableId;
+  options: Options[];
+  onClick: (option: any) => void;
+}
+
+interface Options {
+  id: number;
+  display_name: string;
+}
+
+const TEN_OPTIONS_HEIGHT = 10 * 33;
+
+export function SimpleDataPicker({
+  selectedEntity,
+  options,
+  onClick,
+}: SimpleDataPickerProps) {
+  const shouldShowSearchBar = options.length > 10;
+  const [searchText, setSearchText] = useState("");
+  function filterSearch(option: Options): boolean {
+    if (searchText) {
+      return normalizeString(option.display_name).includes(
+        normalizeString(searchText),
+      );
+    }
+
+    return true;
+  }
+
+  return (
+    <Paper w="300px" p="sm">
+      {shouldShowSearchBar ? (
+        <TextInput
```

Comment:
Use the same component as the entity picker

https://github.com/metabase/metabase/blob/3b7e9e77611ee620992d0b46edaa0695db4673b5/frontend/src/metabase/common/components/EntityPicker/components/EntityPickerModal/EntityPickerModal.tsx#L387-L395

![image](https://github.com/user-attachments/assets/febe70f9-ed98-44a8-a9d9-c45e5e5fd072)